apps/agent/README.md

```diff
@@ -0,0 +1,128 @@
+# Agent
+
+
+
+## 1. Mechanism
+
+## 2. Architecture
```

Comment:
I think I would put notes on the architecture later (perhaps in an appendix or etc. section), as I don't think this is as immediately interesting as the entry points and external interfaces.